BudgetSquirrel.Api/ClientApp/src/app/shared/services/account.service.ts

```diff
@@ -39,6 +39,7 @@ export class AccountService {
   }
 
   public logout(): Observable<boolean> {
+    console.log("Test");
```

Comment:
Hi Test.

Reply:
I can remove that was playing around with the logout stuff.